### DISCUSSION TITLE
Using database outside container

### DISCUSSION THREAD

**Discussion by visualcohol** [docker-library/ghost#408] [General]
I'm trying to make a containerized Ghost instance on a Linux which uses the hosts database.

I tried to create an extra host in composer file, defining it as "database__connection__host" but after starting the container I get host error in logs. Tried to connect to host DB from a test Ubuntu container via host.docker.internal and worked flawlessly.

Did anybody encounter this scenario, and can point me in the right direction? tx

![image](https://github.com/docker-library/ghost/assets/13293752/04620352-615d-4640-8958-86a6ea8a5e81)

**Comment by tianon** [comment]
I think `host.docker.internal` is a Docker Desktop specific thing: :sweat_smile:

```console
$ docker run --rm tianon/network-toolbox dig host.docker.internal +short
$ 
```

Again on my DD-using Windows machine:

```console
$ docker run --rm tianon/network-toolbox dig host.docker.internal +short
192.168.XX.YY
$ 
```

(I also see it in your `extra_hosts`, but I don't think that supports hostnames on the right-hand-side, so I hope that `host-gateway` is replaced with an actual IP address in your real file? :eyes:)
